cluster/addons/fluentd-gcp/fluentd-gcp-configmap.yaml

```diff
@@ -243,47 +252,37 @@ data:
     # TODO(random-liu): Keep this for compatibility, remove this after
     # cri container runtime rolls out.
     <source>
-      type systemd
+      @type systemd
       filters [{ "_SYSTEMD_UNIT": "docker.service" }]
       pos_file /var/log/gcp-journald-docker.pos
       read_from_head true
       tag docker
     </source>
 
     <source>
-      type systemd
+      @type systemd
       filters [{ "_SYSTEMD_UNIT": "{{ container_runtime }}.service" }]
       pos_file /var/log/gcp-journald-container-runtime.pos
       read_from_head true
       tag container-runtime
     </source>
 
     <source>
-      type systemd
+      @type systemd
       filters [{ "_SYSTEMD_UNIT": "kubelet.service" }]
       pos_file /var/log/gcp-journald-kubelet.pos
       read_from_head true
       tag kubelet
     </source>
 
     <source>
-      type systemd
+      @type systemd
       filters [{ "_SYSTEMD_UNIT": "node-problem-detector.service" }]
       pos_file /var/log/gcp-journald-node-problem-detector.pos
       read_from_head true
       tag node-problem-detector
     </source>
   monitoring.conf: |-
-    # Prometheus monitoring
```

Comment:
Just FYI, the `google-fluentd` package by default comes with these Prometheus monitoring configurations already, which is why they are removed from here.

Reply:
Comment was left at the top of this PR, but it got disassociated from the relevant commit after the rebase.

Reply:
Oh Github...